static/app/components/events/featureFlags/eventFeatureFlagList.tsx

```diff
@@ -91,13 +99,18 @@ export function EventFeatureFlagList({
   });
 
   const hasFlagContext = Boolean(event.contexts?.flags?.values);
+  const anyEventHasContext =
```

Comment:
For readability below, could we flip this to `noEventHasFlagContext`?

Reply:
hmm i wanted to keep the pattern as `!hasFlagContext`  on the line below 

https://github.com/getsentry/sentry/blob/6afab8985ce56f808b0e193b647093572117ff2a/static/app/components/events/featureFlags/eventFeatureFlagList.tsx#L112